### PR TITLE
LP-2281 Fix all linting errors for openedx/features/cms

### PR DESCRIPTION
**Pull request by aliassad**
### Description

[LP-2281](https://philanthropyu.atlassian.net/browse/LP-2281)

### Notes

Ran isort first and added missing doc-strings for modules and methods.

- `constants.py` added module doc-string
- `helpers.py` added module doc-string
- `tests/factories.py` added module, methods doc-strings
- `tests/helpers.py` added module doc-string removed unused code and fix lambda(deprecated-lambda) expression
- `cms/tests/test_helpers.py` removed unused params and added missing doc-strings
- `tests/test_views.py ` fixed import order
- `cms/urls.py` added missing doc-strings
- `cms/views.py` added missing doc-strings and fixed exception handling pylint issue

-------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10)

### CODE REVIEW

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

File: openedx/features/cms/tests/helpers.py [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
-from openedx.features.cms import helpers
-from common.djangoapps.student.roles import CourseInstructorRole, CourseStaffRole
 from cms.djangoapps.contentstore.tests.test_courseware_index import COURSE_CHILD_STRUCTURE
-from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory
+from common.djangoapps.student.roles import CourseInstructorRole, CourseStaffRole
 from lms.djangoapps.courseware.courses import get_course_by_id
+from openedx.features.cms import helpers
+from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory
+
+from .factories import CourseRerunFactory
```
Comment (muak50) [inline openedx/features/cms/tests/helpers.py:15]: Verify imports sorting with isort.

**Review by muak50 — APPROVED** [review]
LGTM 👍

File: openedx/features/cms/tests/helpers.py (hunk 1 above)
Comment (aliassad) [reply openedx/features/cms/tests/helpers.py:15]: yes it was done and here too

> **pylint:** ✔️
> **pycodestyle:** ✔️
> **iSort:** ✔️

**Review by ahmed-faraz46 — APPROVED** [review]
LGTM 👍

**Pull request merged**